net/host_test.go

```diff
@@ -71,6 +75,9 @@ func newHost(t *testing.T, port uint16) *host {
 
 	h, err := NewHost(cfg)
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		_ = h.Stop()
```

Comment:
Unless there are known times when we expect this to give an error, I'd change it to:
```
assert.NoError(t, h.Stop())
```
Using `assert` instead of `require` above, because you would want the test to continue.